tern/utils/container.py

```diff
@@ -35,7 +35,7 @@
 except IOError:
     logger.critical("Docker daemon not running")
     raise Exception("Critical Error using Docker API. See logs for details")
-except OSError:
+except OSError:  # pylint: disable=duplicate-except
```

Comment:
I wonder why pylint caught this as a duplicate-except. Is it because they raise the same general exception? If that's the case we can fix it by raising IOError and OSError with that sentence.

Reply:
Haven't dug into it, but it could be that IOError and OSError are not dups on Windows?